config.js

```diff
@@ -11,12 +11,142 @@ System.config({
 System.config({
```

Comment:
`config.js` is mostly generated by `jspm install`. If there's no custom mappings in this config, I'd suggest removing this file from the repo and `.gitignore`-ing this file.